### PR TITLE
[WIP] Cancel bounties reason 

### PR DESCRIPTION
**Pull request by octavioamu**
##### Checklist

- [ ] linter status: 100% pass
- [ ] changes don't break existing behavior
- [ ] commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)

##### Refers/Fixes

Refs: #2948

### CODE REVIEW

File: app/assets/v2/js/pages/kill_bounty.js [hunk 1]
```diff
@@ -24,9 +24,31 @@ window.onload = function() {
...
 
+        const selectedRadio = $('input[name=canceled_bounty_reason]:checked').val();
+        let reasonCancel;
+
+        if (selectedRadio == 'other') {
+          reasonCancel = $('#reason_text').val();
+        } else {
+          reasonCancel = selectedRadio;
+        }
+        var payload = {
```
Comment (thelostone-mc) [inline app/assets/v2/js/pages/kill_bounty.js]: update to `const` ?

File: app/dashboard/views.py [hunk 2]
```diff
@@ -1247,7 +1294,6 @@ def extend_issue_deadline(request):
     }
     return TemplateResponse(request, 'extend_issue_deadline.html', context)
 
-
```
[the remark this answers is not in the document]
Comment (octavioamu) [reply app/dashboard/views.py]: 😅

File: app/assets/v2/js/pages/kill_bounty.js (hunk 1 above)
Comment (thelostone-mc) [reply app/assets/v2/js/pages/kill_bounty.js]: *let ? :P

Comment (octavioamu) [reply app/assets/v2/js/pages/kill_bounty.js]: Sorry I confused with line 33, my bad, you are right

**Comment by octavioamu** [comment]
❤️ Thanks @thelostone-mc , you rock!

**Pull request merged**